src/_gettsim/parameters/sozialv_beitr.yaml

```
@@ -881,6 +889,12 @@ mindestlohn:
     scalar: 10.45
   2022-10-01:
     scalar: 12
+  2024-01-01:
+    scalar: 12.41
+    reference: https://www.bundesregierung.de/breg-de/themen/arbeit-und-soziales/mindestlohn-steigt-2223632
+  2025-01-01:
+    scalar: 12.82
+    reference: https://www.bundesregierung.de/breg-de/themen/arbeit-und-soziales/mindestlohn-steigt-2223632
```

source is [Vierte Verordnung zur Anpassung der Höhe des Mindestlohns (Vierte Mindestlohnanpassungsverordnung – MiLoV4) Vom 24. November 2023](https://www.recht.bund.de/bgbl/1/2023/321/VO.html)

```suggestion
  2024-01-01:
    scalar: 12.41
    reference: V. v. 24.11.2023 BGBl. 2023 I Nr. 321
  2025-01-01:
    scalar: 12.82
    reference: V. v. 24.11.2023 BGBl. 2023 I Nr. 321
```

---

Thanks, didn't find that one